bin/node/runtime/src/lib.rs

```diff
@@ -569,6 +569,7 @@ impl pallet_staking::Config for Runtime {
 	type OnStakerSlash = NominationPools;
 	type WeightInfo = pallet_staking::weights::SubstrateWeight<Runtime>;
 	type BenchmarkingConfig = StakingBenchmarkingConfig;
+	type MaxRewardPoints = ConstU32<300>;
```

Comment:
The number of keys in this map is the number of validators -- staking should have a single `MaxValidators`, and use that as the bound for this type as well.

Reply:
was wondering about that, thanks!